src/material/list/_list-theme.scss

```diff
@@ -33,6 +33,10 @@
       background: mat-color($background, 'hover');
     }
   }
+
+  .mat-selection-list-single-select .mat-list-option.mat-selected {
+    background: mat-color($background, hover, 0.12);
```

Comment:
Wouldn't this make it difficult to distinguish between selected and hovered options?

Reply:
This is the same style used in MatSelect.

It actually is darker than the hover color, despite how the line reads.

![single-select-hover](https://user-images.githubusercontent.com/923677/71993175-a48a3380-3204-11ea-8b78-a89bead6d36c.png)